packages/sqlite/src/index.ts

```diff
@@ -1,8 +1,10 @@
-import * as sqlite from 'sqlite3';
 import {escapeSQLiteIdentifier} from '@databases/escape-identifier';
 import sql, {SQLQuery, isSqlQuery, FormatConfig} from '@databases/sql';
 import Mutex from './Mutex';
-const Queue = require('then-queue');
+const DatabaseConstructor = require("better-sqlite3");
+type Database = typeof DatabaseConstructor;
+type Statement = typeof DatabaseConstructor.Statement;
+type DatabaseOptions = typeof DatabaseConstructor.BetterSqlite3Options;
```

Comment:
These are all just `any` since you're using `require(...)`. If you want the actual types, you'll need to do something like:

```
import Database, {Statement, DatabaseOptions} from 'better-sqluite3'
```

Reply:
I tried the above and it didn't really work. I got:

```
FAIL   node  packages/sqlite/src/__tests__/stream.test.ts
  ● Test suite failed to run

    packages/sqlite/src/index.ts:3:8 - error TS1259: Module '"/Users/matteo/Repositories/atdatabases/node_modules/@types/better-sqlite3/index"' can only be default-imported using the 'esModuleInterop' flag

    3 import Database, {Statement, DatabaseOptions} from 'better-sqlite3';
             ~~~~~~~~

      node_modules/@types/better-sqlite3/index.d.ts:156:1
        156 export = Database;
            ~~~~~~~~~~~~~~~~~~
        This module is declared with using 'export =', and can only be used with a default import when using the 'esModuleInterop' flag.
    packages/sqlite/src/index.ts:3:30 - error TS2305: Module '"better-sqlite3"' has no exported member 'DatabaseOptions'.

    3 import Database, {Statement, DatabaseOptions} from 'better-sqlite3';
                                   ~~~~~~~~~~~~~~~

 FAIL   node  packages/sqlite/src/__tests__/index.test.ts
  ● Test suite failed to run

    packages/sqlite/src/index.ts:3:8 - error TS1259: Module '"/Users/matteo/Repositories/atdatabases/node_modules/@types/better-sqlite3/index"' can only be default-imported using the 'esModuleInterop' flag

    3 import Database, {Statement, DatabaseOptions} from 'better-sqlite3';
             ~~~~~~~~

      node_modules/@types/better-sqlite3/index.d.ts:156:1
        156 export = Database;
            ~~~~~~~~~~~~~~~~~~
        This module is declared with using 'export =', and can only be used with a default import when using the 'esModuleInterop' flag.
    packages/sqlite/src/index.ts:3:30 - error TS2305: Module '"better-sqlite3"' has no exported member 'DatabaseOptions'.

    3 import Database, {Statement, DatabaseOptions} from 'better-sqlite3';
```

If I add `esModuleInterop: true`, then I get (as an example):

```
    packages/sqlite/src/index.ts:28:31 - error TS2709: Cannot use namespace 'Database' as a type.

    28   private readonly _database: Database;
```